src/css/typography.css

```diff
@@ -1,9 +1,9 @@
 @import "fonts.css";
 
 :root {
-	--font-size-50: 0.685 rem;
-	--font-size-75: 0.75 rem;
-	--font-size-100: 0.875 rem;
+	--font-size-50: 0.685rem;
+	--font-size-75: 0.75rem;
+	--font-size-100: 0.875rem;
 	--font-size-125: 1 rem;
```

Comment:
Do you have to remove the space here as well?

Reply:
Ope, don't know how that snuck past me. Yes, great catch! Fixed in [0a62071](https://github.com/harvard-lil/capstone-static/pull/48/commits/0a620710872362e600fceec447eb9d1a0d1ab89f)